services/grpc/lightning/lightning.methods.js

```diff
@@ -237,6 +237,19 @@ async function ensurePeerConnected(payload = {}) {
   return this.connectPeer({ addr: payload })
 }
 
+/**
+ * Estimates on-chain fee
+ *
+ * @param {string} address
+ * @param {number} amount amount in satoshis
+ * @param {number} targetConf desired confirmation time
+ * @returns {Object} {fee_sat, feerate_sat_per_byte}
```

Comment:
Either `@returns {Promise<Object>}` or `@returns {Promise<EstimateFeeResponse>}` (`EstimateFeeResponse` is the type returned by lnd https://api.lightning.community/#estimatefee)